libs/matdbg/src/DebugServer.cpp

```diff
@@ -50,13 +50,28 @@ using utils::FixedCapacityVector;
 // serves files directly from the source code tree.
 #define SERVE_FROM_SOURCE_TREE 0
 
-// When set to 1, we will serve an experimental frontend, which will potentially replace the current
-// frontend when ready.
-#define EXPERIMENTAL_WEB_FRAMEWORK 0
+#if SERVE_FROM_SOURCE_TREE
+
+namespace {
+std::string const BASE_URL = "libs/matdbg/web";
+} // anonymous
+
+#else
 
-#if !SERVE_FROM_SOURCE_TREE
 #include "matdbg_resources.h"
-#endif
+#include <unordered_map>
+
+namespace {
+
+struct Asset {
+    utils::CString mime;
+    utils::CString data;
```

Comment:
I wonder if we could use `std::string_view`s here instead of `CString`, since all of this data is constant. This way we avoid copying the data in the `CString` constructor.

Reply:
good call, done